lib/matplotlib/tests/test_backend_pgf.py

```diff
@@ -113,6 +113,7 @@ def test_xelatex():
 
 # test compiling a figure to pdf with pdflatex
 @needs_pdflatex
+@pytest.mark.skipif(not _has_tex_package('ucs'), reason='needs ucs.sty')
```

Comment:
did you mean to remove this line?

Reply:
no?  that test also needs ucs (and it's a single figure test so we can just use the decorator form)